Modules/ImportExcel.md

```diff
@@ -0,0 +1,33 @@
+# Name: ImportExcel
+
+## Description:
+PowerShell module to import/export Excel spreadsheets, without Excel Edit
+
+| Works with | Author(s) | Project URL(s) | Creation Date | Tags |
+|------------|--------|-------------------|---------------|------|
+| PowerShell 4.0 | [Doug Finke](https://twitter.com/dfinke) | [Github](https://github.com/dfinke/ImportExcel) | May 2015 | powershell, excel |
+
+* On the PowerShell Gallery - [ImportExcel](https://www.powershellgallery.com/packages/ImportExcel)
+____
+
+### Shared blog posts:
+* [Introducing the PowerShell Excel Module](https://blogs.technet.microsoft.com/heyscriptingguy/2015/11/25/introducing-the-powershell-excel-module-2/)
+* [Import-Html Use PowerShell to Export Html Tables to Excel](https://dfinke.github.io/2016/import-html-use-powershell-to-export-html-tables-to-excel/)
+* [Read and write Excel spreadsheets with the ImportExcel PowerShell module](https://4sysops.com/archives/read-and-write-excel-spreadsheets-with-the-importexcel-powershell-module/)
+* [The ImportExcel PowerShell Module](https://www.altitude365.com/2016/01/08/importexcel-powershell/)
+____
+
+### Shared videos:
+
```

Comment:
Same as before. Add your name and (b | t)

Also order the subtitles alphabetically. Switch tracking ups with the last one.

Curiosity: why the last has the [image] and the others is empty?